lib/schemaUtils.js

```diff
@@ -1410,7 +1444,8 @@ module.exports = {
       }
       bodyData = bodyObj.example;
       // return example value if present else example is returned
-      if (bodyData.hasOwnProperty('value')) {
+      if (bodyData.hasOwnProperty('value') &&
```

Comment:
@LuisTejedaS Have we varified that this is the only usage like this? I'm not sure but same could be used in other places like in parameters, responses etc.